web/themes/custom/itkdev/itkdev_project_theme/templates/node--project.html.twig

```diff
@@ -14,13 +14,11 @@
  */
 #}
 {% set owner = drupal_entity('user', projectGroup.uid.target_id)['#user'].field_name.value %}
-{% set project_status = [0, 1, 3, 2] %}
-{% set edit_link_text %}
-  <i class="fa-solid fa-pencil"></i>
-{% endset %}
+
+{% set project_status = project_status_evaluation(node.id) %}
 
 <div class="container my-6">
-  <h1 class="inline-block">{{ label }}</h1><a href="{{ url('entity.node.edit_form', {node: 10}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a>
+  <h1 class="inline-block">{{ label }}</h1><a href="{{ url('entity.node.edit_form', {node: node.id}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a>
```

Comment:
We should check if current user can update node before generating the edit URL:

```suggestion
  <h1 class="inline-block">{{ label }}</h1>
  {% if node.access('update') %}
    <a href="{{ url('entity.node.edit_form', {node: node.id}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a>
  {% endif %}
```
(cf. https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityInterface.php/interface/EntityInterface/11.x)